kernel/kernel-config-4.14.x

```diff
@@ -505,8 +510,10 @@ CONFIG_ARCH_WANTS_THP_SWAP=y
 # CONFIG_ZSMALLOC is not set
 CONFIG_GENERIC_EARLY_IOREMAP=y
 CONFIG_ARCH_SUPPORTS_DEFERRED_STRUCT_PAGE_INIT=y
+# CONFIG_DEFERRED_STRUCT_PAGE_INIT is not set
```

Comment:
Are these new disabled options "magic" ones that appeared as a result of enabling memory hotplug, or are they due to a new kernel version?

Wait, did I just spot a `CONFIG_OMG_THIS_ENTIRE_LINUX_KERNEL_CONFIG_THING_HAS_BECOME_ONE_GIGANTIC_HAIR_BALL_OF_CONFUSING_AND_MAGIC_OPTIONS_WHICH_NOBODY_WITHOUT_UBER_NINJA_SKILLS_REALLY_UNDERSTANDS=y`. ... Hmm... maybe not. :smile:

Reply:
@jodh-intel  yes it is sometimes is difficult to know what  options were enabled and what of the rest were automatically added.  
A good point to add PR process for kata. In the commit message desrcribe what kernel config was added explicitly.

Reply:
Yep. But to be clear, I'm not criticizing this PR, I'm moaning about the general way the kernel is configured as it's too complex and "too magic" imho ;)

Reply:
@jodh-intel  yep agree :P